nevergrad/common/test_tools.py

```diff
@@ -84,4 +84,4 @@ def test_sleeper() -> None:
     sleeper.start_timer()
     np.testing.assert_almost_equal(sleeper._get_advised_sleep_duration(), min_sleep, decimal=5)
     sleeper.stop_timer()
-    np.testing.assert_almost_equal(sleeper._get_advised_sleep_duration(), min_sleep, decimal=5)
+    np.testing.assert_almost_equal(sleeper._get_advised_sleep_duration(), min_sleep, decimal=2)
```

Comment:
this should not have been modified, did you modify it AGAIN because of a problem, or is it some artifacts from all diffs that you are passing over?

Reply:
sorry sorry removed